com.unity.render-pipelines.universal/Editor/Deprecated.cs

```diff
@@ -6,3 +10,11 @@ internal class Deprecated
     {
     }
 }
+
+namespace UnityEditor.Rendering.Universal
+{
+    [Obsolete("ForwardRendererDataEditor has been deprecated. Use StandardRendererDataEditor instead (UnityUpgradable) -> StandardRendererDataEditor", true)]
+    [MovedFrom("UnityEditor.Rendering.LWRP")]
+    public class ForwardRendererDataEditor
+    { }
```

Comment:
you need to keep the public members in place.

Also I am not sure if this will not cause an error in the tests due to the 2 updates. Please, run PVS and make sure no ApiUpdater tests fails.

Reply:
Added the `NotSupportedException` and ran PVS (success with the usual warnings).